digital-deck-server/api/controller.js

```diff
@@ -52,6 +57,23 @@ app.post("/new-session", function (req, res) {
   res.send(currSession);
 });
 
+router.post("/draw-cards", async function (req, res) {
```

Comment:
It would be nice if you add some details about this request in the comments above.

Reply:
Sure, I can do that